docs/en/stack/ml/nlp/ml-nlp-deploy-models.asciidoc

```diff
@@ -115,36 +115,48 @@ eland_import_hub_model --url https://<hostname>:<port> --es-api-key <api-key> ..
 --------------------------------------------------
 --
 
-[discrete]
+
 [[ml-nlp-deploy-model]]
 == Deploy the model in your cluster
 
 After you import the model and vocabulary, you can use {kib} to view and manage
 their deployment across your cluster under **{ml-app}** > *Model Management*.
 Alternatively, you can use the
-{ref}/start-trained-model-deployment.html[start trained model deployment API] or
-specify the `--start` option when you run the `eland_import_hub_model` script.
+{ref}/start-trained-model-deployment.html[start trained model deployment API].
+
+You can deploy a model multiple times by assigning a unique deployment ID when 
+starting the deployment. It enables you to have dedicated deployments for 
+different purposes, such as search and ingest. By doing so, you ensure that the 
+search speed remains unaffected by ingest workloads, and vice versa. Having 
+separate deployments for search and ingest mitigates performance issues 
+resulting from interactions between the two, which can be hard to diagnose.
+
+[role="screenshot"]
+image::images/ml-nlp-deployment-id.png["Model deployment on the Trained Models UI."]
+
+It is recommended to fine-tune each deployment based on its specific purpose. To 
+improve ingest performance, increase throughput by adding more allocations to 
+the deployment. For improved search speed, increase the number of threads per 
+allocation.
```

Comment:
Can we add a section on the deployment queues

## Inference request queues

Each allocation of a model deployment has a dedicated queue to buffer inference requests. The size of this queue is controlled by the queue_capacity parameter on the start model deployment API. Once the queue is full any new requests will be rejected until a queued request has been processed freeing capacity in the queue. If multiple ingest pipelines are using the deployment it is possible the queue will fill in which case consider using multiple deployments, increasing the queue capacity or when using bulk upload send a small number of documents at a time.

Inference requests originating from search such as the text_expansion query or the query_vector_builder option in knn search have a higher priority and skip to the front of the queue to be processed before any lower priority requests. The inference ingest processor creates normal priority requests, if the queue contains inference requests from ingest the search requests will be processed first, this makes search response lively at the expense of slowing down ingest which is not a time critical operation. This behaviour is designed to give the best performance where a single deployment is used for both search and ingest as an alternative to creating a dedicated deployment for each.